doc/man7/OSSL_PROVIDER-legacy.pod

```diff
@@ -52,6 +52,30 @@ The OpenSSL legacy provider supports these operations and algorithms:
 
 =back
 
+=head2 Symmetric Ciphers
+
+Not all the symmetric cipher algorithms can be enabled in a particular OpenSSL build.
```

Comment:
_Not all of these symmetric cipher algorithms are enabled by default._

Although, it would also be good to indicate which aren't: _This cipher is not enabled by default.  Use the enable-XXX configuration to enable it._

Reply:
This still needs changing.

Reply:
I added the mentioning to the rc5 section (which seems the only disabled by default). Could you please clarify what else should be done?

Reply:
Change the text to _Not all of these symmetric cipher algorithms are enabled by default_?